README.md

```diff
@@ -8,6 +8,73 @@ so it can directly be executed as a native application on the target platform.
 [![Travis CI](https://api.travis-ci.org/gluonhq/client-maven-plugin.svg?branch=master)](https://travis-ci.org/gluonhq/client-maven-plugin)
 [![BSD-3 license](https://img.shields.io/badge/license-BSD--3-%230778B9.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+### Requirements
+
+#### Mac OS X and iOS
+
+* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-darwin-20.1.0-ea+28.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-darwin-20.1.0-ea+28
+
+* Set `JAVA_HOME` to point to the GraalVM installation directory
+
+For example:
+
+    export JAVA_HOME=$GRAALVM_HOME
+
+##### Additional requirements
+
+* iOS can be built only on Mac OS X
```

Comment:
IMO, this line needs some "emphasis". May be we can use blockquotes(>)  or just bold the entire line?

Reply:
Hmm, not sure we want to highlight this. At some point it won't be a limitation.